clang/test/CodeGen/builtins-elementwise-math.c

```diff
@@ -495,13 +495,13 @@ void test_builtin_elementwise_min(float f1, float f2, double d1, double d2,
 
   // CHECK:      [[CVF1:%.+]] = load <4 x float>, ptr %cvf1, align 16
   // CHECK-NEXT: [[VF2:%.+]] = load <4 x float>, ptr %vf2.addr, align 16
-  // CHECK-NEXT: call <4 x float> @llvm.minnum.v4f32(<4 x float> [[CVF1]], <4 x float> [[VF2]])
+  // CHECK-NEXT: call nsz <4 x float> @llvm.minnum.v4f32(<4 x float> [[CVF1]], <4 x float> [[VF2]])
```

Comment:
I do not know why __builtin_elementwise_min and __builtin_elementwise_max ever handled FP in the first place, but it's not obvious to me that this would choose fmin. Maybe this should stick to using the stronger signed zero behavior?

Reply:
We are planning to add elementwise minnum/maxnum: https://github.com/llvm/llvm-project/pull/129207.